app/containers/HomeContainer/index.js

```diff
@@ -37,71 +37,71 @@ const Container = styled.div`
     padding: ${props => props.padding}px;
   }
 `;
-const RightContent = styled.div`
-  display: flex;
-  align-self: flex-end;
-`;
+// const RightContent = styled.div`
+//   display: flex;
+//   align-self: flex-end;
+// `;
 export function HomeContainer({
-  dispatchGithubRepos,
-  dispatchClearGithubRepos,
+  dispatchArtist,
+  dispatchClearArtistSearch,
   intl,
-  reposData = {},
-  reposError = null,
-  repoName,
+  artistData = {},
+  artistSearchError = null,
+  artistName,
   maxwidth,
   padding
 }) {
   useInjectSaga({ key: 'homeContainer', saga });
   const [loading, setLoading] = useState(false);
 
   useEffect(() => {
-    const loaded = get(reposData, 'items', null) || reposError;
+    const loaded = get(artistData, 'results', null) || artistSearchError;
```

Comment:
const loaded = get(artistData, 'results', artistSearchError) ;

Last parameter of `get` represents the default value